### ISSUE TITLE
AMP accessibility errors

### ISSUE THREAD

**Issue by brentryanjohnson** [GSA/smartpay-website#261]
## ~~Site navigation~~ ✅ Fixed via #259 

- ~~Three navigation button `aria-controls` attributes are causing errors for invalid names (all for missing hyphens)~~
  - How It Works
  - Policies & Audits
  - State Tax Information

### Example
```
<button aria-controls="extended-nav-section-how-it works" aria-expanded="false" class="usa-accordion__button usa-nav__link" type="button"><span>How it Works</span></button>
Note
This BUTTON has an aria-controls attribute value of 'extended-nav-section-how-it works', which includes one or more invalid or duplicate ids
```

## Policies

- Missing `ul` for list items
- `This element does not have a ul element (without an ARIA-assigned role); ol element (without an ARIA-assigned role); or element with a role set to the value: list as a parent`

![Screenshot highlighting a list item in a table](https://github.com/GSA/smartpay-website/assets/32855580/58e0cdb5-829d-4098-b218-5dde77fa0154)

## Site secondary navigation heading

- Secondary (in-page) navigation headings are not sequential
```
<h4/>
Note
This element creates an inappropriate jump in heading levels within the document heading hierarchy
```
![Screenshot showing highlighted h4 heading on left side of page](https://github.com/GSA/smartpay-website/assets/32855580/a55610d4-0333-4cde-a7f6-efb7ff911ca2)
![Screenshot showing highlighted h4 heading on left side of page](https://github.com/GSA/smartpay-website/assets/32855580/7c2e36b0-398f-4d85-9988-b698f4e015ba)

## Video embeds (`iframe`s)

- AMP flags videos with absolute unit declarations for height or width values
  - `This element has a height attribute value in absolute units. Percentage based units need to be used`

> **Note**
> I'm not sure we can do much about this one; videos need a height attribute to display properly, and percentage height gets tricky in this context. I'll dig into how significant of an accessibility issue this is, but neither pa11y nor Lighthouse flagged this one.

## State Tax Forms

- AMP is flagging the `Go` button: `This button element has a suspicious accessible name`

## Summary boxes

- We're using USWDS summary boxes on a few pages (Customer Support; Bank Contact Information)
- This pattern uses `aria-labelledby="summary-box-key-information"`, but that `labelledby` attribute isn't referencing anything

## Logos and Designs

- AMP is flagging the table: `This table appears to be used for layout but does not have a role of presentation`

**Comment by brentryanjohnson** [comment]
☝️ Looking back at the smooth scroll issue, it looks like that accessibility error is caused by one `Purchase` link going to an in-page `id`, and a separate `Purchase` body-text link going to a PDF. I still think we should fix the smooth scroll, but that won't clear the accessibility error. We could add `PDF` to the body link text so the descriptive text is different.

**Comment by brentryanjohnson** [comment]
Smooth scroll issue potentially resolved in #284

**Comment by A-L33** [comment]
Please replace this language:
"Companies include,
AVENTURA TECHNOLOGIES, INC​. Huawei Technologies Company
Subsidiaries or Affiliates of Huawei
ZTE Corporation
Subsidiaries or Affiliates of ZTE Corporation
Hytera Communications Corporation
Subsidiaries or Affiliates of Hytera Communications Corporation
Hangzhou Hikvision Digital Technology Company
Subsidiaries or Affiliates Hangzhou Hikvision Digital Technology Company
Dahua Technology Company
Subsidiaries or Affiliates of Dahua Technology Company
Kaspersky Lab"

With this language:
"[Check our Section 889 tool](https://889.smartpay.gsa.gov/#/) for information about specific businesses."

**Comment by JessicaMarine1** [comment]
@A-L33 @rebekahperillo this is ready for review: https://federalist-ab31a10d-375d-4040-9324-1ae94e8a36b9.sites.pages.cloud.gov/preview/gsa/smartpay-website/staging/policies-and-audits/policies/

**Comment by A-L33** [comment]
Done!
